internal/plugin.go

```diff
@@ -463,6 +463,10 @@ func (p *Plugin) applyDefaults(unmarshaledConfig map[string]interface{}) {
 			}
 		}
 
+		if policy.PruneObjectBehavior == "" {
```

Comment:
You'll also need to handle this at the manifest level (i.e. under `for j := range policy.Manifests {`).

Reply:
So, this is the point I was trying to make in our scrum. If we're passing `pruneObjectBehavior` to `manifests`, we should be passing everything (`remediationAction`, `severity`, `namespaceSelector`, and so on), right?

Reply:
(I've added it into `manifests` for now.)

Reply:
Right. I'm on the fence but I'm leaning towards adding support for those at the manifest level (i.e. ConfigurationPolicy) since we already do for some fields, so the ship has sailed on that so being consistent would be good. If we do go down this path, could you please open a backlog issue for this?

It'd be good to clarify that this would only apply if `consolidateManifests` is `false`.

What do you think?

Reply:
Works for me! I've created the issue: https://github.com/stolostron/backlog/issues/25982